diagnostic.md

```diff
@@ -71,13 +71,13 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
 
  <!-- Answer Starts Here -->
-
+Relative, because you are not using a precise (absolute) file path name to access it, but a path relative to the current project directory.
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
-
+Either can be done, but the safer way to do it would be through an absolute path in case of any changes to folders shifted around etc.
```

Comment:
This is correct, but in most cases you would probably use a relative path because it is easier and an absolute path is not necessary.